app.py

```diff
@@ -15,17 +15,21 @@
 import joblib
 import gdown
 import os 
+from flasgger import Swagger, LazyJSONEncoder
+from flasgger import swag_from
 
-from tensorflow import keras
 
 from ml_lib_remla.preprocessing import Preprocessing
 
+from config import swagger_template, swagger_config
+from constants import GDRIVE_ID
```

Comment:
Isn't it better to set an environment variable for this instead? Env variable will allow us to change the model without building a new docker image

Reply:
Yeah this is already an issue, but for this release this is more convenient. 
I hope u are ok with that ^